### PR TITLE
Add socks-proxy to hive3.1 and increase memory limits

### PR DESCRIPTION
**Pull request by homar**

### CODE REVIEW

File: testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh [hunk 1]
```diff
@@ -1 +1,28 @@
...
+# if you want this to be tweakable on a per-user basis
+# export HADOOP_MAPRED_HOME=/usr/lib/hadoop-mapreduce
+
+# The maximum amount of heap to use, in MB. Default is 1000.
 export HADOOP_HEAPSIZE=256
+
+# Extra Java runtime options.  Empty by default.
+export HADOOP_NAMENODE_OPTS="$HADOOP_NAMENODE_OPTS -Xmx512m"
+export YARN_OPTS="$YARN_OPTS -Xmx256m"
+export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Xmx512m"
```
Comment (findepi) [inline testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh:28]: I don't see this in `testing/hdp3.1-hive/files/etc/hadoop/conf/hadoop-env.sh`. Why this line?

**Review by findepi — APPROVED** [review]
One question.

For both commits, add info where this come from (eg "Copied from hdp3.1")

File: testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh (hunk 1 above)
Comment (homar) [reply testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh:28]: Without it some map reduce jobs triggered by hive queries died with `java heap out of memory` exception. I tried several options and did small research

Comment (findepi) [reply testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh:31]: let's add this nugget of knowledge as a code comment

Comment (findepi) [reply testing/hive3.1-hive/files/opt/hadoop/etc/hadoop/hadoop-env.sh:31]: you already did, awesome

**Pull request merged**